src/html-media-source.js

```diff
@@ -133,6 +133,8 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     // capture the associated player when the MediaSource is
     // successfully attached
     this.on('sourceopen', (event) => {
+      // TODO: find a better way to get the player this MediaSoruce
+      // is attached to
```

Comment:
There isn't a better way unless they change MSE :smile: you might as well remove the comment.